src/api/node-api.ts

```diff
@@ -7,10 +7,16 @@ class NodeApi {
 
     public async findAllNodesOfType(assetType?: string): Promise<ContentNodeTransport[]> {
         return httpClientV2.get(`/package-manager/api/nodes?assetType=${assetType}`)
-            .catch(e=> {
+            .catch(e => {
                 throw new FatalError(`Problem getting nodes of type ${assetType}: ${e}`);
             });
     }
+
+    public async findOneByKeyAndRootNodeKey(packageKey: string, nodeKey: string): Promise<ContentNodeTransport | null> {
+        return httpClientV2.get(`/package-manager/api/nodes/${packageKey}/${nodeKey}`).catch(e => {
+            return null
```

Comment:
Why do we return null here?

Reply:
Because there are cases when we cant find a node and it throws a 404 but for us that is a null value

Reply:
In this case do we want to continue? As i see from the code we are trying to access an id of a null object in this case. Will this throw an error?